packages/health/ios/Classes/HealthDataTypes.swift

```diff
@@ -0,0 +1,241 @@
+import HealthKit
+
+enum HealthDataTypes: String, CaseIterable {
+  case ACTIVE_ENERGY_BURNED
+  case BASAL_ENERGY_BURNED
+  case BLOOD_GLUCOSE
+  case BLOOD_OXYGEN
+  case BLOOD_PRESSURE_DIASTOLIC
+  case BLOOD_PRESSURE_SYSTOLIC
+  case BODY_FAT_PERCENTAGE
+  case BODY_MASS_INDEX
+  case BODY_TEMPERATURE
+  case DISTANCE_WALKING_RUNNING
+  case ELECTRODERMAL_ACTIVITY
+  case EXERCISE_TIME
+  case FLIGHTS_CLIMBED
+  case FORCED_EXPIRATORY_VOLUME
+  case HEART_RATE
+  case HEART_RATE_VARIABILITY_SDNN
+  case HEIGHT
+  case HIGH_HEART_RATE_EVENT
+  case IRREGULAR_HEART_RATE_EVENT
+  case LOW_HEART_RATE_EVENT
+  case MINDFULNESS
+  case RESTING_HEART_RATE
+  case SLEEP_ASLEEP
+  case SLEEP_AWAKE
+  case SLEEP_IN_BED
+  case STEPS
+  case WAIST_CIRCUMFERENCE
+  case WALKING_HEART_RATE
+  case WATER
+  case WEIGHT
+  case WORKOUT
+
+  case DIETARY_CAFFEINE
+  case DIETARY_CALCIUM
+  case DIETARY_CARBS_CONSUMED
+  case DIETARY_CHOLESTEROL
+  case DIETARY_COPPER
+  case DIETARY_ENERGY_CONSUMED
+  case DIETARY_FAT_MONOUNSATURATED
+  case DIETARY_FAT_POLYUNSATURATED
+  case DIETARY_FAT_SATURATED
+  case DIETARY_FATS_CONSUMED
+  case DIETARY_FIBER
+  case DIETARY_FOLATE
+  case DIETARY_IRON
+  case DIETARY_MAGNESIUM
+  case DIETARY_MANGANESE
+  case DIETARY_NIACIN
+  case DIETARY_PANTOTHENIC_ACID
+  case DIETARY_PHOSPHORUS
+  case DIETARY_POTASSIUM
+  case DIETARY_PROTEIN_CONSUMED
+  case DIETARY_RIBOFLAVIN
+  case DIETARY_SELENIUM
+  case DIETARY_SODIUM
+  case DIETARY_SUGAR
+  case DIETARY_THIAMIN
+  case DIETARY_VITAMIN_A
+  case DIETARY_VITAMIN_B12
+  case DIETARY_VITAMIN_B6
+  case DIETARY_VITAMIN_C
+  case DIETARY_VITAMIN_D
+  case DIETARY_VITAMIN_E
+  case DIETARY_VITAMIN_K
+  case DIETARY_WATER
+  case DIETARY_ZINC
+
+  static let nutrientsToWrite: [HealthDataTypes] = HealthDataTypes.allCases.filter { $0.rawValue.hasPrefix("DIETARY") }
```

Comment:
In Swift `static let/var` or global scopes are implicitly `lazy` (`late`). Plus, they live on the stack rather than the heap.